xio-core/src/test/java/com/xjeffrose/xio/fixtures/OkHttpUnsafe.java

```diff
@@ -32,15 +32,6 @@
     return keyManagerFactory.getKeyManagers();
   }
 
-  public static KeyManager[] getEmptyKeyManager() throws Exception {
```

Comment:
this function is unused and a duplicate

Reply:
This whole class can be removed and the one in xio-test can be used instead.